src/content/docs/emoji-key.md

```diff
@@ -22,18 +37,18 @@ Emoji/Type | Represents | Comments |
 🎨 <br /> `design` | Design | Links to the logo/iconography/visual design/etc. |
 💡 <br /> `example` | Examples | Links to the examples |
 📋 <br /> `eventOrganizing` | Event Organizers | Links to event page |
-💵 <br /> `financial` | Financial Support | People or orgs who provide financial support, links to relevant page |
-🔍 <br /> `fundingFinding` | Funding/Grant Finders | People who help find financial support |
-🤔 <br /> `ideas` | Ideas & Planning | |
-🚇 <br /> `infra` | Infrastructure | Hosting, Build-Tools, etc. Links to source file (like `travis.yml`) in repo, if applicable |
-🚧 <br /> `maintenance` | Maintenance | People who help in maintaining the repo, links to commits by the user on this project |
-🧑‍🏫 <br /> `mentoring` | Mentoring | People who mentor new contributors, links to the repo home |
+💵 <br /> `financial` | Financial Support | People or orgs who provide financial support, links to relevant page
+🔍 <br /> `fundingFinding` | Funding/Grant Finders | People who help find financial support
+🤔 <br /> `ideas` | Ideas & Planning | Open up issues |
```

Comment:
This is from @reshamas PR to update the table!! i cherry picked the commit in and added it here 🚀